modules/microshift-about-remote-health-monitoring.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// microshift_support/microshift-remote-cluster-monitoring.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-about-remote-health-monitoring_{context}"]
+= About remote health monitoring with {microshift-short}
+
+Remote health monitoring is conducted in {microshift-short} by the collection of telemetry and configuration data about your cluster and reports it to Red Hat with the Telemeter API. A cluster that reports data to Red{nbsp}Hat through Telemetry is considered a _connected cluster_.
```

Comment:
"...and configuration data about your cluster and it is reported to Red Hat with the Telemeter API."